.travis/BuildLinux.sh

```diff
@@ -19,6 +19,8 @@ export PATH=$PATH:/work/texlive/bin/x86_64-linux
 
 ccache -z
 
+# We don't need debug symbols during CI and we are able to reduce memory
```

Comment:
"during CI and we are" -> "during CI, so, by turning off the symbols, we are"

(same in MacOsBuild.sh)

Reply:
Marking this comment as resolved because it was deemed too nitpicky.